### PR TITLE
HPCC-27322 log access host/instance fields 

### PR DESCRIPTION
**Pull request by rpastrana**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
Dependent on several other PRs

**Comment by rpastrana** [comment]
@jakesmith this is dependent on another PR (https://github.com/hpcc-systems/HPCC-Platform/pull/15837) with substantial number of changes, and therefore not worth reviewing this one yet...
but this PR exposes logaccess filters based on "instance" as you requested, and also by "host" which would be useful in baremetal. Please provide you thoughts.
BTW, I do plan on exposing a free-hand filter mechanism which would allow caller to filter by field name.

**Review by jakesmith — CHANGES_REQUESTED** [review]
@rpastrana - looks ok, apart from left in tracing.

File: system/logaccess/ElasticStack/ElasticStackLogAccess.cpp [hunk 1]
```diff
@@ -132,6 +136,48 @@ ElasticStackLogAccess::ElasticStackLogAccess(const std::vector<std::string> &hos
...
+toXML(&logMap, xml);
+DBGLOG("$$$$logMap: %s", xml.str());
+            DBGLOG(">>>>>Processing 'instance': '%s'", xml.str());
+            if (logMap.hasProp(LOGMAP_INDEXPATTERN_ATT))
+                m_instanceIndexSearchPattern = logMap.queryProp(LOGMAP_INDEXPATTERN_ATT);
+            if (logMap.hasProp(LOGMAP_SEARCHCOL_ATT))
+            {
+                DBGLOG("Found instance %s -- m_instanceSearchColName current value: '%s'", LOGMAP_SEARCHCOL_ATT, m_instanceSearchColName.str());
+                m_instanceSearchColName = logMap.queryProp(LOGMAP_SEARCHCOL_ATT);
+                DBGLOG("m_instanceSearchColName new value: '%s'", m_instanceSearchColName.str());
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply system/logaccess/ElasticStack/ElasticStackLogAccess.cpp:151]: Yep, these snuck in. Removing all trace messages.

**Review by jakesmith — APPROVED** [review]
@rpastrana - looks good

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**